content/en/container.md

```diff
@@ -28,3 +28,11 @@ There are limitations, however.
 Since containers share the same operating system, processes can be considered less secure than alternatives. 
 Containers also require limits on the shared resources. 
 To guarantee resources, administrators must constrain and limit memory and CPU usage so that other applications do not perform poorly.
+
+## Further Reading
```

Comment:
`## Further Reading` may not be a necessary header, but I added it to separate the `In sign language` section from the content above.

Reply:
Yes, I would skip further reading an use In sign language as H2